Code/GraphMol/MolDraw2D/DrawMolMCHCircleAndLine.cpp

```diff
@@ -203,6 +203,7 @@ void DrawMolMCHCircleAndLine::calcSymbolEllipse(unsigned int atomIdx,
                                                 Point2D &centre,
                                                 double &xradius,
                                                 double &yradius) const {
+  PRECONDITION(atomIdx < atCds_.size() && atomIdx < atomLabels_.size(), "")
```

Comment:
I will approve as-is, but it's not terrible to actually use that final argument to explain the failure. ;-)